### DISCUSSION TITLE
How to properly configure a CreateReactApp v5 app so that Jest doesn't complain when using react-markdown?

### DISCUSSION THREAD

**Discussion by nulltoken** [remarkjs/remark#939] [Q&A] [answered]
👋 Following https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#im-having-problems-with-esm-and-create-react-app recommendation, I've logged https://github.com/facebook/create-react-app/issues/11946 

I'm starting this discussion here with the hope that someone among the community would have a hint/workaround in order to not suffer from the "Must use import to load ES module" error message from a Jest test.

Note: I'd rather _not_ mock `react-markdown`.

Help? :wink:

**Comment by ChristianMurphy** [comment]
There are a few options noted in https://github.com/remarkjs/react-markdown/issues/635
specifically https://github.com/remarkjs/react-markdown/issues/635#issuecomment-956158474 seems to be commonly used

**Comment by nulltoken** [comment]
@ChristianMurphy Thanks but it looks like those solutions target CRA 4.x (and as mentioned above, I'd really like to avoid the mocking approach)

**Comment by ChristianMurphy** [comment, marked as answer]
> Thanks but it looks like those solutions target CRA 4.x

I'd give them a try all the same,
and would also look into https://github.com/facebook/create-react-app/issues/11946#issuecomment-1045155651
